google-cloud-recommender-bom/pom.xml

```diff
@@ -64,17 +64,27 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-recommender</artifactId>
-        <version>0.4.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-recommender:current} -->
+        <version>1.0.0</version><!-- {x-version-update:google-cloud-recommender:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-recommender-v1</artifactId>
+        <version>1.0.0</version><!-- {x-version-update:grpc-google-cloud-recommender-v1:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>grpc-google-cloud-recommender-v1beta1</artifactId>
-        <version>0.4.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-recommender-v1beta1:current} -->
+        <version>0.5.0</version><!-- {x-version-update:grpc-google-cloud-recommender-v1beta1:current} -->
```

Comment:
should this be kept as v1beta1?

Reply:
Yeah, the v1beta1 artifacts should stay at 0.x